### PR TITLE
EOS-15385_fixed m0_filesystem_stats to run without parameter

### PR DESCRIPTION
**Pull request by rajatpatil98**
previously m0_filesystem_stats was giving error, when we run without parameter.
Made correction in m0_filesystem_stats, Now we can run m0_filesystem_stats without passing any parameter.

Signed-off-by: Rajat Patil <rajat.r.patil@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by atulsdeshmukh2312 — APPROVED** [review]

File: utils/spiel/m0_filesystem_stats [hunk 1]
```diff
@@ -153,7 +153,7 @@ fi
 
 get_hostname()
 {
-        host=" "
+        host=""
```
Comment (huanghua78) [inline utils/spiel/m0_filesystem_stats:156]: Good catch!

Please always be careful with empty string and the "test -z".

File: utils/spiel/m0_filesystem_stats [hunk 2]
```diff
@@ -206,13 +206,13 @@ main()
...
 	if [ -z "$profile_addr" ] ; then
-		profile_addr=$($consul_query_cmd | grep profiles: | sed -n -e 's/^.*\(profiles:\)/\1/p'  | awk '{print $1}'| sed 's/^[^:]*://' | sed 's/:/,/')
+		profile_addr=$($consul_query_cmd | grep profiles | sed -n -e 's/^.*\(profiles\)/\1/p'  | awk '{print $1}' | cut -d'/' -f2 | cut -d':' -f1,2 | sed 's/:/, /g')
 	fi
 
 	if [ -z "$client_addr" ] ; then
 		local cmd="tail -2"
 		[[  $systemd == 1 ]] && cmd="tail -1"
-		client_addr=$(get_endpoints_of m0_client_other | $cmd | head -1)
+		client_addr=$(get_endpoints_of m0_client_other | $cmd)
```
Comment (huanghua78) [inline utils/spiel/m0_filesystem_stats:215]: Why is this changed?
If the last "head -1" is not needed, then, if the "tail -2", this will give 2 lines.
Please test it.

**Comment by chandradharraval** [comment]
@rajatpatil98,
Have we addressed all above comments and is it ready to merge?

**Comment by madhavemuri** [comment]
@rajatpatil98 : Can you check it with client pods ?

**Comment by rajatpatil98** [comment]
@madhavemuri @chandradharraval  : Previously even with these changes m0_filesystem_stats was not working, there was some internal issue not only in this script.
I will check this again and update.

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete
